naucse/templates/_base.html

```diff
@@ -84,6 +84,10 @@
                         Licence: <a href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International</a> není-li uvedeno jinak
                     </p>
                 {% endif %}
+
+                <p>
+                    <a href="{{ url_for("course", course="course/meta") }}">Jak přidat kurz</a>
+                </p>
```

Comment:
Myslím že není potřeba mít odkaz na metakurz na každé stránce.
Můžeš to dát třeba jen na úvodní stránku?